### PR TITLE
Adjust React and React Native versions to current app

### PR DESCRIPTION
**Pull request by karauda**

### CODE REVIEW

**Comment by singhleo** [comment]
We should make a migration script helper for developers who are still using old packages in their custom code.

**Comment by singhleo** [comment]
let's merge it only on when app releases.

**Comment by singhleo** [comment]
@karauda we shouldn't change the package.json version as it's just a template and customers might want it from 1.0.0

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "buddyboss-custom-code",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (singhleo) [inline package.json:3]: Keep it 1.0.0 would be best.

**Review by singhleo — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (karauda) [reply package.json:3]: ok will change

**Comment by karauda** [comment]
> @karauda we shouldn't change the package.json version as it's just a template and customers might want it from 1.0.0

@singhleo this is adjusting to the current master version, not to 0.70.10 yet. this is outdated already so we should merge it earlier.

**Pull request merged**